ChangeLog

```diff
@@ -1,3 +1,16 @@
+2021-01-28  Iñaki Ucar  <iucar@fedoraproject.org>
+
+        * DESCRIPTION (Version, Date): Roll minor version
+        * inst/include/Rcpp/config.h: Idem
+
+    * inst/include/Rcpp/iostream/Rstreambuf.h: Support for global Rcout/Rcerr by
+    adding `-DRCPP_USE_GLOBAL_ROSTREAM` to `PKG_CPPFLAGS`. This behavior mimics
+    std::cout/cerr. Required initialization code is automatically generated.
+    * inst/include/Rcpp/routines.h: Ditto
+    * src/api.cpp: Ditto
+    * src/attributes.cpp: Ditto
+    * src/rcpp_init.cpp: Ditto
```

Comment:
If you have another pass over the files, maybe adjust whitespace here.  In Emacs I sometimes mark a region and run `M-x untabify` over it.  I can also also do it later....    I guess in the old days this was a simple eight-space tab but these days most of us prefer spaces. The [editorconfig](https://editorconfig.org) file may interfer too, maybe there is a better setting for `ChangeLog` files to.

(And we probably don't want to spuriously modify the whole file so don't whack it with `sed` or alike.)